docsrc/content/index.md

```diff
@@ -13,7 +13,12 @@ The Fraud Structured Streaming Proofshop is a targeted event to prove structured
 
 ### What is included?
 
-A Powerpoint presentation will walk the prospect through DSE Spark Structured Streaming and the use case.
+This asset includes the following:
+ - A Structured Streaming application that takes data from X and does XYZ.
+ - A Powerpoint presentation that walks users through DSE Spark Structured Streaming code and the use case
+ - Kafka
+ - Notebooks that show X Y and Z
+ - Web application on port 8081 that does X Y Z
 
 
 ### Business Take Aways
```

Comment:
I'd love to see a couple of sentences in this section that delve into the fraud use case since we don't really address it anywhere else in the docs.